packages/urql/updates/index.js

```diff
@@ -1,90 +1,129 @@
-import createComment from './createComment'
+import { get } from 'lodash/fp'
+import meQuery from '@hylo/graphql/queries/meQuery'
 import meCheckAuthQuery from '@hylo/graphql/queries/meCheckAuthQuery'
+import makeAppendToPaginatedSetResolver from './makeAppendToPaginatedSetResolver'
 
 export default {
   Mutation: {
-    addProposalVote: (result, args, cache, info) => { 
+    addProposalVote: (result, args, cache, info) => {
+      // TODO: URQL - This will result in Me being fully re-queried with every new skilled added, which may be fine...
```

Comment:
Is this comment directed at a different mutation? Sounds like you are referring to skills, but this is about proposalVotes, which I would expect to impact a Post (and linked collections) but not Me/currentUser?

Reply:
Ah yes, sorry. Was copy/pasted. Yes, when this one invalidates it will almost certainly trigger a re-query of the entire current group { posts } set. But it's usually fine and isn't as bad as it sounds as it happens in the background. Just wanted to point it out. And we will get these more refined soon, I finally am feeling confident of that, and there will be better examples in here to reference.